css-positioning/absolute.css

```diff
@@ -0,0 +1,44 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: relative;
+  bottom: 27rem;
+  left: 14rem;
```

Comment:
Instead of doing really huge box offsets, try switching to `position: absolute` and use offsets from the sides you want to be close to.